pkg/netutil/netutil.go

```diff
@@ -0,0 +1,11 @@
+package netutil
+
+import "net"
+
+func DialUnix(network, address string) (*net.UnixConn, error) {
+	addr, err := net.ResolveUnixAddr(network, address)
```

Comment:
Why do we need to resolved the address? Why not create a net.UnixAddr, doing basically what the code we want to replace did?

Reply:
Because that's what is done elsewhere. This is called out in the commit message now.